addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1050,9 +1050,14 @@ registry.ScrollButton = registry.anchorSlide.extend({
      */
     _onAnimateClick: function (ev) {
         ev.preventDefault();
-        const $nextElement = this.$el.closest('section').next();
-        if ($nextElement.length) {
-            this._scrollTo($nextElement);
+        // Scroll to the first visible section after the current section.
+        const currentSectionEl = this.$target[0].closest('section');
+        const sectionEls = [...currentSectionEl.parentNode.querySelectorAll('section')];
+        const currentSectionIndex = sectionEls.indexOf(currentSectionEl);
+        const nextSectionEls = sectionEls.slice(currentSectionIndex + 1);
+        const nextEl = nextSectionEls.find((el) => window.getComputedStyle(el).display !== 'none');
```

Comment:
For this one, I'd like to use `is(':visible')` of jQuery (when we'll get rid of jQuery in a future master, we'll make an util which use their definition: `!!(elem.offsetWidth || elem.offsetHeight || elem.getClientRects().length)`, that's probably more efficient and more precise (considers 0x0 elements invisible))